python/plugins/processing/tests/testdata/gdal_algorithm_raster_tests.yaml

```diff
@@ -363,6 +363,28 @@ tests:
         hash: 30409eb496900df4ceeab37200a91552c350dbc7761eb089dd75a329
         type: rasterhash
 
+  - algorithm: gdal:rasterize
```

Comment:
Instead of/as well as this test a command line test should be added to https://github.com/qgis/QGIS/blob/master/python/plugins/processing/tests/GdalAlgorithmsRasterTest.py#L1573

(The command line tests are much more robust and easy to debug vs hash based tests like this)

Reply:
Test added also to GdalAlgorithmsRasterTest.py: https://github.com/qgis/QGIS/pull/47205/commits/cf4502dd20a69aaff7397828962aca57f33ce83e.
I apologize for force-pushing.